sdk/documenttranslation/azure-ai-documenttranslation/README.md

```diff
@@ -1,35 +1,402 @@
+                TranslationTarget(target_url=target_container_sas_url_fr, glossaries=glossaries, language_code="fr"),
+            ],
+        )
+    ]
+)  # type: JobStatusResult
+
+job_result = document_translation_client.wait_until_done(job.id)  # type: JobStatusResult
+
+print("Job created on: {}".format(job_result.created_on))
+print("Job last updated on: {}".format(job_result.last_updated_on))
+print("Total number of translations on documents: {}".format(job_result.documents_total_count))
+
+print("Of total documents...")
+print("{} failed".format(job_result.documents_failed_count))
+print("{} succeeded".format(job_result.documents_succeeded_count))
+print("{} in progress".format(job_result.documents_in_progress_count))
+print("{} not yet started".format(job_result.documents_not_yet_started_count))
+print("{} cancelled".format(job_result.documents_cancelled_count))
+
+if job_result.status == "Succeeded":
+    print("Our translation job succeeded")
+
+if job_result.status == "Failed":
+    print("All documents failed in the translation job")
+
+# check document statuses... see next sample
+```
+
+### Check status on individual documents
+Check status and translation progress of each document.
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.documenttranslation import DocumentTranslationClient
+
+endpoint = "https://<resource-name>.cognitiveservices.azure.com/"
+credential = AzureKeyCredential("<api_key>")
+job_id = "<job-id>"
+
+document_translation_client = DocumentTranslationClient(endpoint, credential)
+
+docs_to_retry = []
+
+documents =  document_translation_client.list_all_document_statuses(job_id)  # type: ItemPaged[DocumentStatusResult]
+
+for doc in documents:
+    if doc.status == "Succeeded":
+        print("Document at {} was translated to {} language".format(
+            doc.translated_document_url, doc.translate_to
+        ))
+    if doc.status == "Running":
+        print("Document ID: {}, translation progress is {} percent".format(
+            doc.id, doc.translation_progress*100
+        ))
+    if doc.status == "Failed":
+        print("Document ID: {}, Error Code: {}, Message: {}".format(
+            doc.id, doc.error.code, doc.error.message
+        ))
+        if doc.source_url not in docs_to_retry:
+            docs_to_retry.append(doc.source_url)
+
+```
+
+### List translation jobs
+Enumerate over the translation jobs submitted for the resource.
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.documenttranslation import DocumentTranslationClient
+
+endpoint = "https://<resource-name>.cognitiveservices.azure.com/"
+credential = AzureKeyCredential("<api_key>")
+
+document_translation_client = DocumentTranslationClient(endpoint, credential)
+
+jobs = document_translation_client.list_submitted_jobs()  # type: ItemPaged[JobStatusResult]
```

Comment:
is this snippet runnable in Python? if so, consider changing the sample to list 10 jobs only or something like that

Reply:
Not running.